### PR TITLE
add caas-relevant changes

### PR DESCRIPTION
**Pull request by bavarianbidi**
add some caas specific changes to build ubuntu 20.04

### CODE REVIEW

File: images/capi/ansible/roles/providers/tasks/vmware.yml [hunk 1]
```diff
@@ -43,22 +43,18 @@
...
+#    url:  https://raw.githubusercontent.com/vmware/cloud-init-vmware-guestinfo/master/install.sh
+#    dest: /tmp/cloud-init-vmware.sh
+#
+#- name: Execute cloud-init-vmware.sh
+#  shell: bash -o errexit -o pipefail /tmp/cloud-init-vmware.sh
+#
+#- name: Remove cloud-init-vmware.sh
+#  file:
+#    path:  /tmp/cloud-init-vmware.sh
+#    state: absent
```
Comment (chrischdi) [inline images/capi/ansible/roles/providers/tasks/vmware.yml]: to reduce the diff:
```suggestion
#- name: Download cloud-init datasource for VMware Guestinfo
#  get_url:
#    url:  '{{ guestinfo_datasource_script }}'
#    dest: /tmp/cloud-init-vmware.sh
#    mode: 0700
#
#- name: Execute cloud-init-vmware.sh
#  shell: bash -o errexit -o pipefail /tmp/cloud-init-vmware.sh
#  environment:
#    REPO_SLUG: '{{ guestinfo_datasource_slug }}'
#    GIT_REF:   '{{ guestinfo_datasource_ref }}'
#
#- name: Remove cloud-init-vmware.sh
#  file:
#    path:  /tmp/cloud-init-vmware.sh
#    state: absent
```

File: images/capi/ansible/roles/sysprep/tasks/main.yml [hunk 2]
```diff
@@ -153,15 +142,6 @@
...
   when: ansible_memory_mb.swap.total != 0
 
-- name: Create a log directory for auditd in Photon OS and Amazon Linux 2
-  file:
-    path: "/var/log/audit"
-    state: directory
-    mode: "0640"
-    owner: root
-    group: root
-  when: kubernetes_semver != '' and (ansible_os_family == "VMware Photon OS" or ansible_distribution == "Amazon") # Ensure audit log is not created for HA Proxy images, since this task is shared amongst build targets
```
Comment (chrischdi) [inline images/capi/ansible/roles/sysprep/tasks/main.yml]: Could we keep it to reduce the diff?

File: images/capi/ansible/roles/dhc/tasks/main.yaml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+---
+- import_tasks: debian.yml
```
Comment (sbueringer) [inline images/capi/ansible/roles/dhc/tasks/main.yaml:2]: seems like we can drop the whole file :)

**Review by chrischdi — COMMENTED** [review]
/lgtm

**Review by sbueringer — APPROVED** [review]

File: images/capi/ansible/roles/sysprep/tasks/main.yml (hunk 2 above)
Comment (bavarianbidi) [reply images/capi/ansible/roles/sysprep/tasks/main.yml]: sure

**Review by chrischdi — APPROVED** [review]

**Comment by sbueringer** [comment]
@bavarianbidi side note: should we also push the caas-1910 branch in this repo? (just so we don't have to switch to the other repo / keep the renamed repo in case we want to build based on our old version)

**Pull request merged**